src/engine/controls/bpmcontrol.h

```diff
@@ -165,6 +171,7 @@ class BpmControl : public EngineControl {
 
     // m_pBeats is written from an engine worker thread
     mixxx::BeatsPointer m_pBeats;
+    mixxx::audio::SampleRate m_sampleRate;
```

Comment:
Please revert. We can't guarantee that this is always up to date. All used variables for calculation need to be atonic.